corehq/apps/case_search/templates/case_search/case_search.html

```diff
@@ -82,17 +90,21 @@ <h3>{% trans "Search Parameters" %}</h3>
                 <span class="btn btn-outline-primary" id="add-input" data-bind="click: addParameter"><i class="fa fa-plus"></i> {% trans "Add input" %}</span>
               </div>
             </div>
-            <div class="card card-default">
+            <div class="card card-default mb-3">
               <div class="card-header">
                 <strong>XPath Expression</strong>
               </div>
               <div class="card-body">
-                <div class="form-group" style="width:100%" data-bind="foreach: xpath_expressions" >
-                  <textarea style="min-width: 50%" spellcheck="false" class="form-control vertical-resize"
+                <div class="row align-items-center" style="width:100%" data-bind="foreach: xpath_expressions" >
+                  <div class="col-9">
```

Comment:
We usually target a medium screen size, so this would be `col-md-9`, so I'd do that for consistency's sake. (I searched through the bootstrap docs for a few min before finding that this targets extra small screens.)